Runtime/Model/Metrics/MetricsSubmissionQueue.cs

```diff
@@ -112,12 +112,15 @@ internal void SendPayload(ICollection<T> events, uint attempts = 0)
 
             RequestHandler.Post(SubmissionUrl, payload, (long statusCode, bool httpError, string response) =>
             {
-                if (statusCode == 200)
+                if (httpError == false && statusCode == 200)
```

Comment:
snippet on how we generate each parameter: 

```
var statusCode = request.responseCode;
                var response = request.downloadHandler.text;
                var networkError = request.ReceivedNetworkError();
```

if the status code == 200 there is no way the error will be there. EVen if the error is set, 200 means the server received and accepted the request, so there is no reason to retry it.

Reply:
Agreed. Thanks.